soccer_vision_msgs/CMakeLists.txt

```diff
@@ -12,6 +12,8 @@ find_package(vision_msgs REQUIRED)
 
 rosidl_generate_interfaces(${PROJECT_NAME}
   "msg/Ball.msg"
+  "msg/Obstacle.msg"
+  "msg/Robot.msg"
```

Comment:
I think we should add both a robot and an obstacle array message.

Reply:
Agreed, I think I will put all the arrays in a separate PR.